master/internal/api_workspace.go

```diff
@@ -860,7 +861,7 @@ func (a *apiServer) setWorkspaceNamespaceBindings(ctx context.Context,
 		case metadata.AutoCreateNamespaceAllClusters:
 			if !verified {
 				verified = true
-				license.RequireLicense("auto-create namespace")
+				// license.RequireLicense("auto-create namespace")
```

Comment:
is this still intended to be commented out?

Reply:
I think she's waiting on my updates for the licensing.